packages/fx-core/src/plugins/resource/frontend/dotnet/plugin.ts

```diff
@@ -177,24 +178,25 @@ export class DotnetPluginImpl implements PluginImpl {
   }
 
   public async deploy(ctx: PluginContext): Promise<TeamsFxResult> {
-    Logger.info(Messages.StartDeploy(PluginInfo.displayName));
+    Logger.info(Messages.StartDeploy);
+    await ProgressHelper.startProgress(ctx, DeployProgress);
 
-    this.syncConfigFromContext(ctx);
```

Comment:
When we create config as local variable, shall we remove method of syncConfigFromContext()?

Reply:
The naming could be updated, getConfigFromContext may be better?